diff -rupN pantheon-default-settings.orig/debian/elementary-default-settings.gsettings-override pantheon-default-settings/debian/elementary-default-settings.gsettings-override
--- pantheon-default-settings.orig/debian/elementary-default-settings.gsettings-override	2015-02-13 19:13:57.593295066 +0100
+++ pantheon-default-settings/debian/elementary-default-settings.gsettings-override	2015-02-13 19:18:17.806988752 +0100
@@ -6,9 +6,8 @@ show-day=true
 show-events=false
 
 [org.gnome.desktop.background]
-draw-background=true
 picture-options='zoom'
-picture-uri='file:///usr/share/backgrounds/elementaryos-default'
+picture-uri='file:///usr/share/backgrounds/pantheon/elementaryos-default'
 primary-color='#000000'
 show-desktop-icons=false
 
@@ -22,7 +21,6 @@ font-name='Droid Sans 9'
 gtk-theme='elementary'
 icon-theme='elementary'
 monospace-font-name='Droid Sans Mono 10'
-show-unicode-menu=false
 toolbar-style='icons'
 
 [org.gnome.desktop.screensaver]
diff -rupN pantheon-default-settings.orig/midori/config pantheon-default-settings/midori/config
--- pantheon-default-settings.orig/midori/config	2015-02-13 19:13:57.643293039 +0100
+++ pantheon-default-settings/midori/config	2015-02-13 19:18:37.919640330 +0100
@@ -3,7 +3,7 @@ default-font-family=Droid Sans
 enable-spell-checking=true
 enable-webgl=true
 homepage=http://start.elementaryos.org/
-text-editor=gnome-text-editor %F
+text-editor=scratch-text-editor %F
 show-statusbar=false
 close-buttons-left=true
 first-party-cookies-only=false
diff -rupN pantheon-default-settings.orig/plank/dock1/launchers/softwarecenter.dockitem pantheon-default-settings/plank/dock1/launchers/softwarecenter.dockitem
--- pantheon-default-settings.orig/plank/dock1/launchers/softwarecenter.dockitem	2015-02-13 19:13:57.643293039 +0100
+++ pantheon-default-settings/plank/dock1/launchers/softwarecenter.dockitem	1970-01-01 01:00:00.000000000 +0100
@@ -1,4 +0,0 @@
-[PlankItemsDockItemPreferences]
-Launcher=file:///usr/share/applications/ubuntu-software-center.desktop
-
-
diff -rupN pantheon-default-settings.orig/plank/dock1/settings pantheon-default-settings/plank/dock1/settings
--- pantheon-default-settings.orig/plank/dock1/settings	2015-02-13 19:13:57.643293039 +0100
+++ pantheon-default-settings/plank/dock1/settings	2015-02-13 19:20:31.755913253 +0100
@@ -11,7 +11,7 @@ UnhideDelay=0
 #The monitor number for the dock. Use -1 to keep on the primary monitor.
 Monitor=-1
 #List of *.dockitem files on this dock. DO NOT MODIFY
-DockItems=gala-multitaskingview.dockitem;;midori.dockitem;;geary.dockitem;;maya-calendar.dockitem;;noise.dockitem;;audience.dockitem;;shotwell.dockitem;;switchboard.dockitem;;softwarecenter.dockitem
+DockItems=gala-multitaskingview.dockitem;;midori.dockitem;;geary.dockitem;;maya-calendar.dockitem;;noise.dockitem;;audience.dockitem;;shotwell.dockitem;;switchboard.dockitem
 #The position for the dock on the monitor.
 Position=3
 #The dock's position offset from center (in percent).
