diff -rupN retroarch.orig/.git/FETCH_HEAD retroarch/.git/FETCH_HEAD
--- retroarch.orig/.git/FETCH_HEAD	1970-01-01 01:00:00.000000000 +0100
+++ retroarch/.git/FETCH_HEAD	2018-06-18 15:15:23.328877310 +0200
@@ -0,0 +1,3 @@
+ca3b8149a80a7b3802f19104a8f6c3feb415d307		branch 'master' of /home/alucryd/packages/src/retroarch
+ca3b8149a80a7b3802f19104a8f6c3feb415d307	not-for-merge	branch 'coverity_scan' of /home/alucryd/packages/src/retroarch
+f3f79c1a4a64e2d9ac7ea220aa27caaa6561d297	not-for-merge	branch 'index_parse' of /home/alucryd/packages/src/retroarch
diff -rupN retroarch.orig/retroarch.cfg retroarch/retroarch.cfg
--- retroarch.orig/retroarch.cfg	2018-06-18 15:12:44.907736627 +0200
+++ retroarch/retroarch.cfg	2018-06-18 15:15:23.232209948 +0200
@@ -750,7 +750,7 @@
 
 # Assets directory. This location is queried by default when menu interfaces try to look for
 # loadable assets, etc.
-# assets_directory =
+assets_directory = /usr/share/retroarch/assets
 
 # Dynamic wallpapers directory. The place to store the wallpapers dynamically
 # loaded by the menu depending on context.
@@ -763,10 +763,10 @@
 # rgui_browser_directory =
 
 # Core directory for libretro core implementations.
-# libretro_directory =
+libretro_directory = /usr/lib/libretro
 
 # Core info directory for libretro core information.
-# libretro_info_path =
+libretro_info_path = /usr/share/libretro/info
 
 # Path to content database directory.
 # content_database_path =
@@ -784,7 +784,7 @@
 # audio_filter_dir =
 
 # Defines a directory where shaders (Cg, CGP, GLSL) are kept for easy access.
-# video_shader_dir =
+video_shader_dir = /usr/share/libretro/shaders
 
 # Recording output directory. Where recordings are saved.
 # recording_output_directory =
