diff -rupN sqlheavy-0.1.1.orig/configure sqlheavy-0.1.1/configure
--- sqlheavy-0.1.1.orig/configure	2012-04-30 08:48:23.000000000 +0200
+++ sqlheavy-0.1.1/configure	2013-02-27 20:08:56.294226442 +0100
@@ -12322,101 +12322,6 @@ $as_echo "yes" >&6; }
 fi
 
 
-VALA_PACKAGE=libvala-0.16
-
-pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for VALA" >&5
-$as_echo_n "checking for VALA... " >&6; }
-
-if test -n "$VALA_CFLAGS"; then
-    pkg_cv_VALA_CFLAGS="$VALA_CFLAGS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"\$VALA_PACKAGE >= \$VALA_REQUIRED\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "$VALA_PACKAGE >= $VALA_REQUIRED") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_VALA_CFLAGS=`$PKG_CONFIG --cflags "$VALA_PACKAGE >= $VALA_REQUIRED" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-if test -n "$VALA_LIBS"; then
-    pkg_cv_VALA_LIBS="$VALA_LIBS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"\$VALA_PACKAGE >= \$VALA_REQUIRED\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "$VALA_PACKAGE >= $VALA_REQUIRED") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_VALA_LIBS=`$PKG_CONFIG --libs "$VALA_PACKAGE >= $VALA_REQUIRED" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-   	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        VALA_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "$VALA_PACKAGE >= $VALA_REQUIRED" 2>&1`
-        else
-	        VALA_PKG_ERRORS=`$PKG_CONFIG --print-errors "$VALA_PACKAGE >= $VALA_REQUIRED" 2>&1`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$VALA_PKG_ERRORS" >&5
-
-	as_fn_error $? "Package requirements ($VALA_PACKAGE >= $VALA_REQUIRED) were not met:
-
-$VALA_PKG_ERRORS
-
-Consider adjusting the PKG_CONFIG_PATH environment variable if you
-installed software in a non-standard prefix.
-
-Alternatively, you may set the environment variables VALA_CFLAGS
-and VALA_LIBS to avoid the need to call pkg-config.
-See the pkg-config man page for more details." "$LINENO" 5
-
-elif test $pkg_failed = untried; then
-     	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-	{ { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "The pkg-config script could not be found or is too old.  Make sure it
-is in your PATH or set the PKG_CONFIG environment variable to the full
-path to pkg-config.
-
-Alternatively, you may set the environment variables VALA_CFLAGS
-and VALA_LIBS to avoid the need to call pkg-config.
-See the pkg-config man page for more details.
-
-To get pkg-config, see <http://pkg-config.freedesktop.org/>.
-See \`config.log' for more details" "$LINENO" 5; }
-
-else
-	VALA_CFLAGS=$pkg_cv_VALA_CFLAGS
-	VALA_LIBS=$pkg_cv_VALA_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-
-fi
-
-
-
 
 pkg_failed=no
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GTK" >&5
@@ -12608,7 +12513,7 @@ done
 
 # Checks for library functions.
 
-ac_config_files="$ac_config_files Makefile data/Makefile data/schemas/Makefile data/schemas/profiling/Makefile docs/Makefile examples/Makefile gtk/Makefile gtk/sqlheavygtk-$SQLHEAVY_API_VERSION.deps sqlheavy/Makefile sqlheavy-$SQLHEAVY_API_VERSION.pc sqlheavygtk-$SQLHEAVY_API_VERSION.pc utils/Makefile"
+ac_config_files="$ac_config_files Makefile data/Makefile data/schemas/Makefile data/schemas/profiling/Makefile docs/Makefile examples/Makefile gtk/Makefile gtk/sqlheavygtk-$SQLHEAVY_API_VERSION.deps sqlheavy/Makefile sqlheavy-$SQLHEAVY_API_VERSION.pc sqlheavygtk-$SQLHEAVY_API_VERSION.pc"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -13642,7 +13547,6 @@ do
     "sqlheavy/Makefile") CONFIG_FILES="$CONFIG_FILES sqlheavy/Makefile" ;;
     "sqlheavy-$SQLHEAVY_API_VERSION.pc") CONFIG_FILES="$CONFIG_FILES sqlheavy-$SQLHEAVY_API_VERSION.pc" ;;
     "sqlheavygtk-$SQLHEAVY_API_VERSION.pc") CONFIG_FILES="$CONFIG_FILES sqlheavygtk-$SQLHEAVY_API_VERSION.pc" ;;
-    "utils/Makefile") CONFIG_FILES="$CONFIG_FILES utils/Makefile" ;;
 
   *) as_fn_error $? "invalid argument: \`$ac_config_target'" "$LINENO" 5;;
   esac
diff -rupN sqlheavy-0.1.1.orig/configure.ac sqlheavy-0.1.1/configure.ac
--- sqlheavy-0.1.1.orig/configure.ac	2012-04-30 08:40:31.000000000 +0200
+++ sqlheavy-0.1.1/configure.ac	2013-02-27 20:07:48.980086732 +0100
@@ -69,11 +69,6 @@ AC_SUBST(GMODULE_LIBS)
 PKG_CHECK_MODULES(SQLITE3, sqlite3 >= $SQLITE3_REQUIRED)
 AC_SUBST(SQLITE3_CFLAGS)
 AC_SUBST(SQLITE3_LIBS)
-VALA_PACKAGE=libvala-0.16
-PKG_CHECK_MODULES(VALA, $VALA_PACKAGE >= $VALA_REQUIRED)
-AC_SUBST(VALA_PACKAGE)
-AC_SUBST(VALA_CFLAGS)
-AC_SUBST(VALA_LIBS)
 PKG_CHECK_MODULES(GTK, gtk+-2.0 >= $GTK_REQUIRED)
 GTK_PACKAGE=gtk+-2.0
 AC_SUBST(GTK_PACKAGE)
@@ -114,6 +109,5 @@ AC_CONFIG_FILES([Makefile
                  gtk/sqlheavygtk-$SQLHEAVY_API_VERSION.deps
                  sqlheavy/Makefile
                  sqlheavy-$SQLHEAVY_API_VERSION.pc
-                 sqlheavygtk-$SQLHEAVY_API_VERSION.pc
-                 utils/Makefile])
+                 sqlheavygtk-$SQLHEAVY_API_VERSION.pc])
 AC_OUTPUT
diff -rupN sqlheavy-0.1.1.orig/Makefile.am sqlheavy-0.1.1/Makefile.am
--- sqlheavy-0.1.1.orig/Makefile.am	2012-04-30 08:31:00.000000000 +0200
+++ sqlheavy-0.1.1/Makefile.am	2013-02-27 20:07:16.363019724 +0100
@@ -4,7 +4,6 @@ SUBDIRS = \
 	sqlheavy \
 	gtk \
 	examples \
-	utils \
 	docs \
 	data \
 	$(NULL)
diff -rupN sqlheavy-0.1.1.orig/Makefile.in sqlheavy-0.1.1/Makefile.in
--- sqlheavy-0.1.1.orig/Makefile.in	2012-04-30 08:48:24.000000000 +0200
+++ sqlheavy-0.1.1/Makefile.in	2013-02-27 20:10:07.781416589 +0100
@@ -288,7 +288,6 @@ SUBDIRS = \
 	sqlheavy \
 	gtk \
 	examples \
-	utils \
 	docs \
 	data \
 	$(NULL)
