diff -rupN wxWidgets-3.0.0.orig/src/gtk/toplevel.cpp wxWidgets-3.0.0/src/gtk/toplevel.cpp
--- wxWidgets-3.0.0.orig/src/gtk/toplevel.cpp	2013-11-11 14:10:39.000000000 +0100
+++ wxWidgets-3.0.0/src/gtk/toplevel.cpp	2014-01-13 22:59:18.431922311 +0100
@@ -38,6 +38,9 @@
     #include <gdk/gdkx.h>
     #include <X11/Xatom.h>  // XA_CARDINAL
     #include "wx/unix/utilsx11.h"
+    #endif
+#ifdef GDK_WINDOWING_WAYLAND 
+    #include <gdk/gdkwayland.h>
 #endif
 
 #include "wx/gtk/private.h"
@@ -730,7 +733,7 @@ bool wxTopLevelWindowGTK::Create( wxWind
             m_gdkDecor |= GDK_DECOR_TITLE;
 #if defined(GDK_WINDOWING_WAYLAND) && GTK_CHECK_VERSION(3,10,0)
         else if (
-            GDK_IS_WAYLAND_SCREEN(gtk_window_get_screen(GTK_WINDOW(m_widget))) &&
+            GDK_IS_WAYLAND_DISPLAY(gtk_widget_get_display(m_widget)) &&
             gtk_check_version(3,10,0) == NULL)
         {
             gtk_window_set_titlebar(GTK_WINDOW(m_widget), gtk_header_bar_new());
@@ -1546,4 +1549,4 @@ bool wxTopLevelWindowGTK::CanSetTranspar
     return XQueryExtension(gdk_x11_get_default_xdisplay (),
                            "Composite", &opcode, &event, &error);
 #endif
-}
+}
\ No newline at end of file
