=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2014-07-28 00:43:13 +0000
+++ CMakeLists.txt	2014-08-20 21:26:19 +0000
@@ -5,7 +5,8 @@
 cmake_policy (VERSION 2.8)
 
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-
+option (NO_INDICATOR_NG "Disable support for Indicators NG" OFF)
+option (OLD_LIB_IDO "Compile for LibIDO version 13.04 or previous" OFF)
 #
 # Base bits
 #
@@ -80,6 +81,14 @@
 include(ValaVersion)
 ensure_vala_version("0.16" MINIMUM)
 
+if (NO_INDICATOR_NG)
+    set(EXTRA_VALA_OPTS ${EXTRA_VALA_OPTS} -D NO_INDICATOR_NG)
+endif()
+
+if (OLD_LIB_IDO)
+    set(EXTRA_VALA_OPTS ${EXTRA_VALA_OPTS} -D OLD_LIB_IDO)
+endif()
+
 include(ValaPrecompile)
 vala_precompile (VALA_C ${EXEC_NAME}
     src/WingpanelApp.vala
@@ -114,6 +123,7 @@
     -g
 	--thread
     --vapidir=${CMAKE_CURRENT_SOURCE_DIR}/vapi/
+    ${EXTRA_VALA_OPTS}
 )
 
 # Settings Schema

=== modified file 'src/Indicator/IndicatorFactory.vala'
--- src/Indicator/IndicatorFactory.vala	2013-09-14 21:26:39 +0000
+++ src/Indicator/IndicatorFactory.vala	2014-08-20 21:26:19 +0000
@@ -49,8 +49,10 @@
         // Legacy indicator libraries
         load_indicators_from_dir (Build.INDICATORDIR, true, skip_list);
 
+#if !NO_INDICATOR_NG
         // Ng indicators
         load_indicators_from_dir ("/usr/share/unity/indicators", false, skip_list);
+#endif
     }
 
     private void load_indicators_from_dir (string dir_path, bool legacy_libs_only, string skip_list) {
@@ -79,17 +81,21 @@
         try {
             Indicator.Object indicator = null;
 
+#if !NO_INDICATOR_NG
             if (legacy_lib) {
+#endif
                 if (!name.has_suffix (".so"))
                     return;
 
                 debug ("Loading Indicator Library: %s", name);
                 indicator = new Indicator.Object.from_file (indicator_path);
+#if !NO_INDICATOR_NG
             } else {
                 debug ("Loading Indicator File: %s", name);
                 indicator = new Indicator.Ng.for_profile (indicator_path, "desktop");
             }
 
+#endif
             if (indicator != null)
                 indicators.add (new IndicatorObject (indicator, name));
             else

=== modified file 'src/WingpanelApp.vala'
--- src/WingpanelApp.vala	2014-07-28 00:43:13 +0000
+++ src/WingpanelApp.vala	2014-08-20 21:26:19 +0000
@@ -35,7 +35,9 @@
     protected override void startup () {
         base.startup ();
 
+#if !OLD_LIB_IDO
         Ido.init ();
+#endif
         Services.EndSessionDialog.register ();
 
         var settings = new Services.Settings ();

