diff -rupN 3dstool-1.0.25.orig/src/romfs.cpp 3dstool-1.0.25/src/romfs.cpp
--- 3dstool-1.0.25.orig/src/romfs.cpp	2017-06-19 23:30:45.261905149 +0200
+++ 3dstool-1.0.25/src/romfs.cpp	2017-06-19 23:32:12.444834329 +0200
@@ -266,7 +266,7 @@ void CRomFs::buildIgnoreList()
 {
 	m_vIgnoreList.clear();
 	m_vRemapIgnoreList.clear();
-	UString sIgnorePath = UGetModuleDirName() + USTR("/ignore_3dstool.txt");
+	UString sIgnorePath = USTR("/usr/share/3dstool/ignore_3dstool.txt");
 	FILE* fp = UFopen(sIgnorePath.c_str(), USTR("rb"));
 	if (fp != nullptr)
 	{
diff -rupN 3dstool-1.1.1.orig/src/ncch.cpp 3dstool-1.1.1/src/ncch.cpp
--- 3dstool-1.1.1.orig/src/ncch.cpp	2018-05-13 22:14:02.488443921 +0200
+++ 3dstool-1.1.1/src/ncch.cpp	2018-05-13 22:15:27.418245089 +0200
@@ -794,7 +794,7 @@ void CNcch::calculateKey()
 
 void CNcch::readExtKey()
 {
-	UString sExtKeyPath = UGetModuleDirName() + USTR("/ext_key.txt");
+	UString sExtKeyPath = USTR("/usr/share/3dstool/ext_key.txt");
 	FILE* fp = UFopen(sExtKeyPath.c_str(), USTR("rb"));
 	if (fp != nullptr)
 	{
@@ -832,7 +832,7 @@ void CNcch::readExtKey()
 
 bool CNcch::writeExtKey()
 {
-	UString sExtKeyPath = UGetModuleDirName() + USTR("/ext_key.txt");
+	UString sExtKeyPath = USTR("/usr/share/3dstool/ext_key.txt");
 	FILE* fp = UFopen(sExtKeyPath.c_str(), USTR("wb"));
 	if (fp == nullptr)
 	{
