=== modified file 'src/ProgressUIHandler.vala'
--- src/ProgressUIHandler.vala	2013-08-16 22:51:30 +0000
+++ src/ProgressUIHandler.vala	2013-11-21 01:08:15 +0000
@@ -64,9 +64,9 @@
     }
 
     private bool server_has_persistence () {
-        unowned List<string> cappabilities = Notify.get_server_caps ();
+        unowned List<string> capabilities = Notify.get_server_caps ();
 
-        return cappabilities.find ("persistence") != null;
+        return capabilities.find ("persistence") != null;
     }
 
     private void progress_info_started_cb (Marlin.Progress.Info info) {

