--- a/src/bamf-matcher.c	2013-05-06 19:16:24.685146000 +0200
+++ b/src/bamf-matcher.c	2013-05-06 19:11:57.877957000 +0200
@@ -2803,30 +2804,30 @@ on_dbus_handle_window_stack_for_monitor
   return TRUE;
 }
 
-static gboolean
-bamf_matcher_has_tab_with_parent_xid (BamfMatcher *matcher, guint64 xid)
-{
-  GList *l;
-  g_return_val_if_fail (BAMF_IS_MATCHER (matcher), FALSE);
-
-  for (l = matcher->priv->views; l; l = l->next)
-    {
-      if (!BAMF_IS_TAB (l->data))
-        continue;
-
-      if (xid == bamf_tab_get_xid (BAMF_TAB (l->data)))
-        return TRUE;
-    }
-
-  return FALSE;
-}
-
 #ifdef HAVE_WEBAPPS
 static void
 on_webapp_child_added (BamfView *application,
                        BamfView *child,
                        gpointer user_data)
 {
+  static gboolean
+  bamf_matcher_has_tab_with_parent_xid (BamfMatcher *matcher, guint64 xid)
+  {
+    GList *l;
+    g_return_val_if_fail (BAMF_IS_MATCHER (matcher), FALSE);
+
+    for (l = matcher->priv->views; l; l = l->next)
+      {
+        if (!BAMF_IS_TAB (l->data))
+          continue;
+
+        if (xid == bamf_tab_get_xid (BAMF_TAB (l->data)))
+          return TRUE;
+      }
+
+    return FALSE;
+  }
+
   GList *l;
   BamfMatcher *self;
   BamfLegacyWindow *legacy_window;
