--- a/data/lightdm-webkit-greeter.conf	2013-03-13 14:59:43.552561800 +0100
+++ b/data/lightdm-webkit-greeter.conf	2013-03-13 15:04:14.765406300 +0100
@@ -8,11 +8,11 @@
 # xft-rgba = Type of subpixel antialiasing (none, rgb, bgr, vrgb or vbgr)
 #
 [greeter]
-background=/usr/share/backgrounds/warty-final-ubuntu.png
-theme-name=Ambiance
-webkit-theme=default
-font-name=Ubuntu 11
+background=usr/share/themes/Adwaita/backgrounds/good-night.jpg
+theme-name=Adwaita
+webkit-theme=webkit
+font-name=Cantarell 11
 xft-antialias=true
 xft-dpi=96
-xft-hintstyle=slight
+xft-hintstyle=hintfull
 xft-rgba=rgb
