diff -rupN switchboard-plug-power.orig/src/Plug.vala switchboard-plug-power/src/Plug.vala
--- switchboard-plug-power.orig/src/Plug.vala	2014-02-20 11:49:24.504118000 +0100
+++ switchboard-plug-power/src/Plug.vala	2014-02-20 11:49:11.347641015 +0100
@@ -128,10 +128,6 @@ namespace Power {
 			grid.attach (scale_label, 0, 0, 1, 1);
 			grid.attach (scale, 1, 0, 1, 1);
 		
-			var lid_closed_box = new ComboBox (_("When the lid is closed:"), @"lid-close-$type-action");
-			grid.attach (lid_closed_box.label, 0, 1, 1, 1);
-			grid.attach (lid_closed_box, 1, 1, 1, 1);
-			
 			if (type != "ac") {
 				var critical_box = new ComboBox (_("When battery power is critically low:"), "critical-battery-action");
 				grid.attach (critical_box.label, 0, 2, 1, 1);
@@ -161,4 +157,4 @@ public Switchboard.Plug get_plug (Module
     debug ("Activating Power plug");
     var plug = new Power.Plug ();
     return plug;
-}
\ No newline at end of file
+}
