diff -rupN noise.orig/data/noise.desktop noise/data/noise.desktop
--- noise.orig/data/noise.desktop	2015-04-13 11:02:23.061729447 +0200
+++ noise/data/noise.desktop	2015-04-13 11:02:45.301129489 +0200
@@ -3,7 +3,7 @@ Type=Application
 Name=Noise
 GenericName=Music Player
 Comment=Listen to music
-Exec=noise %U
+Exec=noise-player %U
 Icon=multimedia-audio-player
 Terminal=false
 Categories=Audio;Music;Player;AudioVideo;GNOME;GTK;
@@ -17,5 +17,5 @@ X-PulseAudio-Properties=media.role=music
 Actions=AboutDialog;
 
 [Desktop Action AboutDialog]
-Exec=noise --about
+Exec=noise-player --about
 Name=About Noise
diff -rupN noise.orig/src/CMakeLists.txt noise/src/CMakeLists.txt
--- noise.orig/src/CMakeLists.txt	2015-04-13 11:02:23.068395935 +0200
+++ noise/src/CMakeLists.txt	2015-04-13 11:03:35.203100073 +0200
@@ -93,7 +93,7 @@ vala_precompile (CLIENT_VALA_C ${SRC_TAR
   ${CLIENT_VAPI}
 )
 
-set (EXEC_NAME ${SRC_TARGET})
+set (EXEC_NAME ${SRC_TARGET}-player)
 
 add_definitions (${DEPS_CFLAGS})
 link_directories (${DEPS_LIBRARY_DIRS})
