=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2015-03-11 07:33:27 +0000
+++ CMakeLists.txt	2015-04-15 10:11:49 +0000
@@ -63,7 +63,6 @@
     libgsignon-glib
     gsignond
     json-glib-1.0
-    webkit2gtk-3.0
     libsoup-2.4
     gmodule-2.0
     rest-0.7
@@ -81,7 +80,6 @@
     libgsignon-glib
     gsignond
     json-glib-1.0
-    webkit2gtk-3.0
     libsoup-2.4
     gmodule-2.0
     rest-0.7
@@ -98,9 +96,19 @@
 # Add all your dependencies to the list below
 pkg_check_modules (DEPS REQUIRED ${DEPS_PKG})
 
-add_definitions (${DEPS_CFLAGS})
-link_libraries (${DEPS_LIBRARIES})
-link_directories (${DEPS_LIBRARY_DIRS})
+# Check for webkit2gtk-4.0 and fallback to webkit2gtk-3.0 if not available
+pkg_check_modules (WEBKIT2GTK webkit2gtk-4.0)
+
+if(WEBKIT2GTK_FOUND)
+  set(DEPS_PACKAGES ${DEPS_PACKAGES} webkit2gtk-4.0)
+else()
+  pkg_check_modules (WEBKIT2GTK REQUIRED webkit2gtk-3.0)
+  set(DEPS_PACKAGES ${DEPS_PACKAGES} webkit2gtk-3.0)
+endif()
+
+add_definitions (${DEPS_CFLAGS} ${WEBKIT2GTK_CFLAGS})
+link_libraries (${DEPS_LIBRARIES} ${WEBKIT2GTK_LIBRARIES})
+link_directories (${DEPS_LIBRARY_DIRS} ${WEBKIT2GTK_LIBRARY_DIRS})
 
 include (ValaPrecompile)
 

