--- a/src/gtk-logout-helper.c	2013-08-23 16:29:47.033397952 +0200
+++ b/src/gtk-logout-helper.c	2013-08-23 17:00:34.797156139 +0200
@@ -29,18 +29,18 @@ with this program.  If not, see <http://
 #include "settings-helper.h"
 
 static void
-consolekit_fallback (LogoutDialogType action)
+systemd_fallback (LogoutDialogType action)
 {
-	g_debug("Falling back to using ConsoleKit for action");
+	g_debug("Falling back to using systemd for action");
 
 	DBusGConnection * sbus = dbus_g_bus_get(DBUS_BUS_SYSTEM, NULL);
 	g_return_if_fail(sbus != NULL); /* worst case */
-	DBusGProxy * proxy = dbus_g_proxy_new_for_name(sbus, "org.freedesktop.ConsoleKit",
-	                                                     "/org/freedesktop/ConsoleKit/Manager",
-	                                                     "org.freedesktop.ConsoleKit.Manager");
+	DBusGProxy * proxy = dbus_g_proxy_new_for_name(sbus, "org.freedesktop.login1",
+	                                                     "/org/freedesktop/login1/Manager",
+	                                                     "org.freedesktop.login1.Manager");
 
 	if (proxy == NULL) {
-		g_warning("Unable to get consolekit proxy");
+		g_warning("Unable to get login1 proxy");
 		return;
 	}
 
@@ -48,20 +48,20 @@ consolekit_fallback (LogoutDialogType ac
 
 	switch (action) {
 		case LOGOUT_DIALOG_TYPE_LOG_OUT:
-			g_warning("Unable to fallback to ConsoleKit for logout as it's a session issue.  We need some sort of session handler.");
+			g_warning("Unable to fallback to systemd for logout as it's a session issue.  We need some sort of session handler.");
 			break;
 		case LOGOUT_DIALOG_TYPE_SHUTDOWN:
-			g_debug("Telling ConsoleKit to 'Stop'");
+			g_debug("Telling systemd to 'PowerOff'");
 			dbus_g_proxy_call(proxy,
-			                  "Stop",
+			                  "PowerOff",
 			                  &error,
 			                  G_TYPE_INVALID,
 			                  G_TYPE_INVALID);
 			break;
 		case LOGOUT_DIALOG_TYPE_RESTART:
-			g_debug("Telling ConsoleKit to 'Restart'");
+			g_debug("Telling systemd to 'Reboot'");
 			dbus_g_proxy_call(proxy,
-			                  "Restart",
+			                  "Reboot",
 			                  &error,
 			                  G_TYPE_INVALID,
 			                  G_TYPE_INVALID);
@@ -74,7 +74,7 @@ consolekit_fallback (LogoutDialogType ac
 	g_object_unref(proxy);
 
 	if (error != NULL) {
-		g_warning("Unable to signal ConsoleKit");
+		g_warning("Unable to signal systemd");
 		g_error_free(error);
 	}
 
@@ -103,7 +103,7 @@ session_action (LogoutDialogType action)
 		g_warning("Unable to get DBus proxy to SessionManager interface: %s", error->message);
 		g_error_free(error);
 
-		consolekit_fallback(action);
+		systemd_fallback(action);
 		return;
 	}		
 	
@@ -132,7 +132,7 @@ session_action (LogoutDialogType action)
 			g_warning ("SessionManager action failed: unknown error");
 		}
 
-		consolekit_fallback(action);
+		systemd_fallback(action);
 	}
 	
 	g_object_unref(sm_proxy);
