--- a/configure.ac	2013-08-23 17:11:27.390525272 +0200
+++ b/configure.ac	2013-08-23 17:12:17.456940576 +0200
@@ -63,8 +63,7 @@ AS_IF([test "x$with_gtk" = x3],
                                        dbus-glib-1
                                        gudev-1.0                                       
                                        gio-unix-2.0
-                                       indicator3-0.4 >= $INDICATOR_REQUIRED_VERSION
-                                       packagekit-glib2)
+                                       indicator3-0.4 >= $INDICATOR_REQUIRED_VERSION)
     ],
     [test "x$with_gtk" = x2],
     [PKG_CHECK_MODULES(SESSIONSERVICE, dbusmenu-glib-0.4 >= $DBUSMENUGLIB_REQUIRED_VERSION
@@ -72,8 +71,7 @@ AS_IF([test "x$with_gtk" = x3],
                                        gudev-1.0                                       
                                        dbus-glib-1
                                        gio-unix-2.0
-                                       indicator-0.4 >= $INDICATOR_REQUIRED_VERSION
-                                       packagekit-glib2)
+                                       indicator-0.4 >= $INDICATOR_REQUIRED_VERSION)
     ]
 )
 
